### PR TITLE
Fix taa py wrapper

### PR DESCRIPTION
**Pull request by krgko**
I tried to use vcx with the sovrin builder network and got an error while starting `faber.py` at step 3. Before fixes, it always returns.
```bash
#1 Provision an agent and wallet, get back configuration details
#2 Initialize libvcx with new configuration
#2.1 Accept transaction author agreement
#3 Create a new schema on the ledger
Traceback (most recent call last):
  File "faber.py", line 194, in <module>
    loop.run_until_complete(main())
  File "/home/user/.pyenv/versions/3.7.6/lib/python3.7/asyncio/base_events.py", line 583, in run_until_complete
    return future.result()
  File "faber.py", line 73, in main
    schema = await Schema.create(f'schema{str(random.randint(1, 101))}', 'degree schema asd', version, ['email', 'first_name', 'last_name'], 0)
  File "/home/user/Desktop/blockchain/indy-sdk/vcx/wrappers/python3/vcx/api/schema.py", line 106, in create
    schema = await Schema._create("vcx_schema_create", constructor_params, c_params)
  File "/home/user/Desktop/blockchain/indy-sdk/vcx/wrappers/python3/vcx/api/vcx_base.py", line 47, in _create
    cls.create.cb)
vcx.error.VcxError: (<ErrorCode.DuplicateSchema: 1088>, {'backtrace': '', 'cause': 'Reject { reason: "client request invalid: InvalidClientTaaAcceptanceError(\\\'Incorrect Txn Author Agreement(digest=273df4edd0756bc4354f587e2f73db01acee711f925fef69bc85f878c3b7dc15) in the request\\\',)" }', 'error': 'Duplicate Schema: Ledger Already Contains Schema For Given DID, Version, and Name Combination', 'message': 'Error: Duplicate Schema: Ledger Already Contains Schema For Given DID, Version, and Name Combination\n  Caused by: Reject { reason: "client request invalid: InvalidClientTaaAcceptanceError(\\\'Incorrect Txn Author Agreement(digest=273df4edd0756bc4354f587e2f73db01acee711f925fef69bc85f878c3b7dc15) in the request\\\',)" }\n'})
```

As tested on python cli
```bash
>>> test_str = "hashed_data"
>>> test_str
'hashed_data'
>>> import json
>>> json.dumps(test_str)
'"hashed_data"'
```

The string from json.dumps() result on `vcx_set_active_txn_author_agreement_meta` will have extra single quote that might have transaction author agreement always incorrect.

After removed json.dumps() from `vcx_set_active_txn_author_agreement_meta` all steps work

### CODE REVIEW

File: vcx/wrappers/python3/demo/faber.py [hunk 1]
```diff
@@ -24,14 +27,15 @@
...
+    'agency_did': 'X2b2m5JyzeroKKB4X44VHY',
+    'agency_verkey': 'HNDzQx2JSytjWMcfqWoTyCtMCyfL7dvKcQfDGvjaVhNY',
+    'wallet_name': 'faber_wallet9',
     'wallet_key': '123',
     'payment_method': 'null',
-    'enterprise_seed': '000000000000000000000000Trustee1',
-    'protocol_type': '3.0',
+    'enterprise_seed': 'bHjydDGBjNBTp5cjGCK8bzbzvW4rR9kb',
+    'protocol_type': '2.0',
+    'communication_protocol': 'aries',
```
Comment (mirgee) [inline vcx/wrappers/python3/demo/faber.py:38]: For me, the TAA works with the original agency provision and dummy cloud agency, which `README.md` directs the user to use. Why this change?

Comment (Patrik-Stas) [reply vcx/wrappers/python3/demo/faber.py:38]: I agree with Mirgee the demo configuration should not have by default have external dependencies (external ledger, 3rd party cloud agent, someone's seed, ...)

**Comment by Patrik-Stas** [comment]
Thanks for contribution @krgko ! 👍 

I don't know if it just me, but it seems like Sovrin BuilderNet is having some issues so it's difficult to verify the changes. I tried to basically port these changes to nodejs demo and run against Sovrin BuilderNet but getting probably unrelated issue:
```
TRACE|indy::services::payments      |           src/services/payments.rs:450 | _handle_constraint <<< result: Err(IndyError { inner:

The requester role "101" doesn't meet to constraint ""2"".

The transaction is not allowed to a requester })
```
@mirgee  any luck testing this?

**Comment by krgko** [comment]
Sorry, I have accidentally added some testing config into PR. Anyways please let me remove this PR and resend PR again. For all of the comments, I will follow up with them.
